db/migrate/20180919211201_add_full_cost_cancellation_to_price_policies.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+class AddFullCostCancellationToPricePolicies < ActiveRecord::Migration[5.0]
+
+  def change
+    add_column :price_policies, :full_cancellation_cost, :boolean, default: false, null: false
```

Comment:
This column name took my brain a few reads to grok fully… I am not sure if any of my suggestions are better, but what do you think about something like:

- `charge_fully_on_cancellation`
- `full_price_cancellation`
- `cancellation_cost_equals_full_cost`

Reply:
Hmm, that's fair that the name isn't clear. I struggled a bit with that at first, because the name could imply an amount rather than a boolean, but ultimately had trouble coming up with something better. Maybe `charge_full_price_on_cancellation`?

Reply:
Oooh, I like that! `charge_full_price_on_cancellation` is good.